.buildkite/pipeline.yml

```diff
@@ -9,6 +9,7 @@ common_params:
   # Common environment values to use with the `env` key.
   - &common_env
     IMAGE_ID: xcode-13 # Remember to address all 'FIXIT-13.1' comments when we'll update to new image
```

Comment:
I guess it's about time we address this, now that we have both 13.3 and 13.4 images available (thanks @transier). I'll look at that in a dedicated PR.